src/Core/Core.csproj

```diff
@@ -34,9 +34,9 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.11.2006.403" />
-    <PackageReference Include="Microsoft.Quantum.CsharpGeneration" Version="0.11.2006.403" />
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.11.2006.403" />
+    <PackageReference Include="Microsoft.Quantum.Compiler" Version="0.12.20062320-beta" />
```

Comment:
Is that a valid build? Looks like the build failed and some providers packages aren't available. Maybe 0.12.20062314-beta instead?

Also, since you're doing this (thank you!), would it make sense to also update the public CI build to minor version 12?\
https://github.com/microsoft/iqsharp/blob/ccadb8ff3b8998bec65e1d60d79eefe078218900/build/ci.yml#L7

Reply:
Ah, normally broken builds don't seem to make it to the alpha feed... thanks for the catch! As for whether to change `appsettings.json`, my only thought was to update everywhere that version number appeared in a uniform fashion, but I'm happy to revert the changes to appsettings.

Reply:
No, sorry, I meant you *should* update appsettings, and I completely missed the fact that you already did. Too late at night to be reading code. :-)